### ISSUE TITLE
Expandable Habitats are only Greenhouses

### ISSUE THREAD

**Issue by Marvball** [BobPalmer/MKS#1210]
Hey there,
is there a general problem with the expandable Habitats of the Tundra-Series? I can´t choose the habitation mode. In the VAB i only can choose  "(greenhouse) => (greenhouse)". And in the Kerbitat Module there´s only the Option "Recycler => Recycler".
There´s a Reddit Post with that problem here https://www.reddit.com/r/KerbalSpaceProgram/comments/58jn1x/usi_mod_question/

Greetings Marvball
(sorry 4 bad english ;)

**Comment by BobPalmer** [comment]
Are you using USILS?

**Comment by Marvball** [comment]
No, i don´t use any Lifesupport Mod. I testet TacLS while searching for the Problem. But after it didn´t work i uninstalled it.

**Comment by Marvball** [comment]
Ah i forgot to say, while my testings i installed a clean new KSP and installed only USI an required Mods. On two PC´s i have the same issue.

**Comment by BobPalmer** [comment]
Then there's no point to the part switching anyway, as it is only relevant for life support.

**Comment by Marvball** [comment]
Oh, i can´t use it as an habitat for 10 Kerbals anymore? Why? That was the best use and make the stations look great :-(

**Comment by BobPalmer** [comment]
That question has nothing to do with what you asked though

**Comment by Marvball** [comment]
OK. I tested it with USI LF and the option to change it is back. Sorry 4 wasting your time. I love your Mods and you do a realy good job!!!

My english is very bad so i don´t understand a lot of the text. So it is not realy easy for me.

My idea was to use this ring as an house for my Kerbals. But everytime i tried to expand i needet any materials. I don´t understand this. I thougt it doesn´t cost materials when it is an habitat. Now i see the opposite. I saw it in an video of my favourite Youtuber. He expanded it for a couple of month with no materials. I don´t know how to get these materials. But it´s ok. I use another part.

So thanks for your work and your time!!! cy
